use-cases/xtclim/preprocessing/preprocess_2d_seasons.py

```diff
@@ -131,7 +131,7 @@ def execute(self):
         ##### 4. Apply to Projection Datasets
 
         for scenario in scenarios:
-            proj_images = np.load(f"input/preprocessed_2d_proj{scenario}_data_allssp.npy")
+            proj_images = np.load(f"input/preprocessed_2d_proj{scenario}_data_allssp.npy", allow_pickle=True)
             proj_time = pd.read_csv("input/dates_proj_data.csv")
```

Comment:
Best practice here would be using pathlib to handle the paths. Same as above comment.